test/system/410-selinux.bats

```diff
@@ -51,18 +51,13 @@ function check_label() {
 }
 
 @test "podman selinux: pid=host" {
-    # FIXME FIXME FIXME: Remove these lines once all VMs have >= 2.146.0
-    # (this is ugly, but better than an unconditional skip)
-    skip_if_no_selinux
+    # FIXME this test fails when run rootless with runc:
+    #   Error: container_linux.go:367: starting container process caused: process_linux.go:495: container init caused: readonly path /proc/asound: operation not permitted: OCI permission denied
     if is_rootless; then
-        if [ -x /usr/bin/rpm ]; then
-            cs_version=$(rpm -q --qf '%{version}' container-selinux)
-        else
-            # SELinux not enabled on Ubuntu, so we should never get here
-            die "WHOA! SELinux enabled, but no /usr/bin/rpm!"
-        fi
+        runtime=$(podman_runtime)
+        test "$runtime" == "crun" \
+            || skip "runtime is $runtime; this test requires crun"
```

Comment:
This is a new skip; I realize that the clutter (removing very-old cruft) makes it hard to review. @rhatdan, do you have any idea why this would fail under runc? More importantly: is this an expected failure?

Reply:
I have no memory of this.  I guess the best we can do is try it out and see if it fails.

Reply:
It _does_ fail -- see the newly-added comment (lines 54-55, green) (and again, I apologize for the difficulty of review. The removed cruft (red) dates to October 2020, when some CI systems had an old version of container-selinux.)

Reply:
podman --runtime=runc run --pid=host alpine cat /proc/self/attr/current
Error: container_linux.go:367: starting container process caused: process_linux.go:495: container init caused: readonly path /proc/asound: operation not permitted: OCI permission denied

Weird
@giuseppe any idea what is going on here?

Reply:
One more data point: on RHEL, IIRC, the failure is in `/proc/bus` instead of `/proc/asound`. This makes me suspect that it's just hitting the alphabetically-first entry under `/proc`. In case that helps.

Reply:
It seems like runc is attempting to write to /proc when --pid=host and SELinux is enabled, and failing.

Reply:
If I build runc from main branch, I get.
```
podman --runtime=$PWD/runc run --pid=host alpine cat /proc/self/attr/current
unconfined_u:system_r:spc_t:s0
```

Reply:
rpm -qf /bin/runc
containerd.io-1.4.3-3.2.fc33.x86_64

Reply:
On my f33 system: `runc-1.0.0-377.rc93.fc33.x86_64`